sdk/communication/Azure.Communication.MediaComposition/api/Azure.Communication.MediaComposition.netstandard2.0.cs

```diff
@@ -145,18 +145,26 @@ public partial class MediaCompositionClient
         public MediaCompositionClient(string connectionString, Azure.Communication.MediaComposition.MediaCompositionClientOptions options) { }
         public MediaCompositionClient(System.Uri endpoint, Azure.AzureKeyCredential keyCredential, Azure.Communication.MediaComposition.MediaCompositionClientOptions options = null) { }
         public MediaCompositionClient(System.Uri endpoint, Azure.Core.TokenCredential tokenCredential, Azure.Communication.MediaComposition.MediaCompositionClientOptions options = null) { }
+        public virtual Azure.Response<Azure.Communication.MediaComposition.MediaCompositionBody> AddInputs(string mediaCompositionId, System.Collections.Generic.IDictionary<string, Azure.Communication.MediaComposition.MediaInput> inputs, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
How do we think about Updates?
We could add a new `UpdateInputs` method, or we change `AddInputs` to `UpsertInputs`, or we could replace Add/Remove with a single `UpdateInputs(IEnumerable<MediaInput> upsert, IEnumerable<string> remove)`.

Reply:
That's a good point, Dom. I like the option of `UpsertInputs`, we still have the granularity but Update + Add fits together well and reflects merge+patch. What are your preferences?

Reply:
Discussed offline, will go with `UpsertInputs` for now